### ISSUE TITLE
Selection of overpressure relief valve

### ISSUE THREAD

**Issue by neelfirst** [RespiraWorks/PhysicalDesign#8]
On patient inhale side, after O2 mixing, before humidification.

TBD: before or after inhale pressure/flow measurement?

**Comment by esc-works** [comment]
Yesterday we purchased a valve from McMaster Carr https://www.mcmaster.com/4277t52-4277T522

**Comment by neelfirst** [comment]
Did we receive this valve and/or test it? Quick slack search doesn't turn anything up.

**Comment by esc-works** [comment]
Yes, sorry! forgot to update. I think this is good as long as we have a fan solution (max flow rate is very low but will reliably prevent high pressures as long as its not like 100 cm H2O). Believe we can close this.

**Comment by neelfirst** [comment]
Confirmed part https://www.mcmaster.com/4277t52-4277T522, closing

**Comment by esc-works** [comment]
This issue is re-opened. Testing the mcmaster part revealed the first one was leaking at about 20-25 cm of pressure. It also had a very small orifice that would have not allowed for real relief anyway. This is annoying but we'll need a better solution.

**Comment by dtgerson** [comment]
We also have to figure out what pressure we want this to relieve at

**Comment by dtgerson** [comment]
see this chain for some valves we decided to try out: https://respiraworks.slack.com/archives/C0137BLF17A/p1590501836181400
